src/test/java/org/medicmobile/webapp/mobile/UtilsTest.java

```diff
@@ -3,15 +3,24 @@
 import android.net.Uri;
 
 import org.junit.*;
+import org.junit.rules.TemporaryFolder;
 import org.junit.runner.*;
 import org.robolectric.*;
 import org.robolectric.annotation.*;
 
-import static org.junit.Assert.*;
+import junit.framework.TestCase;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Optional;
 
 @RunWith(RobolectricTestRunner.class)
 @Config(sdk=28)
-public class UtilsTest {
+public class UtilsTest extends TestCase {
```

Comment:
No special reason, I took it from an article but makes sense to leave it open to extend to other more useful classes. Good learning! I've removed this extension from the other test file as well. 

assertj-core looks very nice, specially now that we've upgraded to java 8 and started to use new features, definitely something to bring up to the `development` channel for everyone knowledge. Specially since we'll be refactoring cht-android and cht-gateway as we go, we'll be adding test coverage (right now is very behind as you can see 😅).